src/app/shared/beta-then-auth-guard-service.ts

```diff
@@ -11,14 +14,24 @@ export class BetaThenAuthService {
 
   constructor(
     private authService: AuthService,
-    private userService: UserService
+    private userService: UserService,
+    private stateService: StateService
   ) { }
 
   canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> {
-    if (this.userService.isBetaEnabled() === true) {
-      return of(true);
-    }
-    return this.authService.canActivateChild(route, state);
+    return iif(
+      () => this.stateService.configuration?._id !== undefined,
+      of(true),
+      this.stateService.couchStateListener('configurations')
+    ).pipe(switchMap((res) => {
+      console.log(`BetaThenAuth: ${res}`);
```

Comment:
Leaving these console log lines in for help with testing. Go ahead and remove before merging if it's ready.